properties.cmake

```diff
@@ -28,20 +31,37 @@ set(CLAW_X2T_CONFIG_PATH "${CMAKE_INSTALL_PREFIX}/etc/")
 
 # Driver files
 set(CLAW_CONF_FILE "claw_f.conf")
+set(CLAW_PY_CONF_FILE "claw_conf.py")
 set(CLAW_COMPILER_FILE "clawfc")
 set(CLAW_LIB_SH "claw_f_lib.sh")
 set(CLAW_X2T_DRIVER_LIB_DIR "${CMAKE_INSTALL_PREFIX}/libexec/")
 
 #
 # OMNI Compiler variables
 #
-set(OMNI_HOME "${CMAKE_INSTALL_PREFIX}")
+set(OMNI_VERSION "1.3.0")
+set(OMNI_GIT_COMMIT_HASH "dd9fbd8a08141e582e83ad8dd5f5267889a23a54" CACHE STRING
+    "OMNI compiler tools GIT commit hash")
```

Comment:
why doe we need a version and a hash, what happen if they are not refering to the same thing ?

Reply:
Right, version is not needed anymore, I'll remove it.